go-controller/pkg/ovn/common.go

```diff
@@ -199,16 +199,20 @@ func (oc *Controller) deletePortGroup(hashName string) {
 	}
 }
 
-func (oc *Controller) getIPFromOvnAnnotation(ovnAnnotation string) string {
-	if ovnAnnotation == "" {
+func (oc *Controller) getMacFromOvnAnnotation(ovnAnnotation string) string {
```

Comment:
@alexanderConstantinescu let's return an error from this function if the MAC doesn't pass `net.ParseMAC()`